lib/internal/errors.js

```diff
@@ -151,6 +152,7 @@ E('ERR_SOCKET_BAD_TYPE',
 E('ERR_SOCKET_CANNOT_SEND', 'Unable to send data');
 E('ERR_SOCKET_BAD_PORT', 'Port should be > 0 and < 65536');
 E('ERR_SOCKET_DGRAM_NOT_RUNNING', 'Not running');
+E('ERR_V8BREAKITERATOR', 'full ICU data not installed. See https://github.com/nodejs/node/wiki/Intl');
```

Comment:
long line here. please line wrap at <= 80 chars.
Running `make lint` should catch these :-)